pkg/controllers/state/metrics.go

```diff
@@ -60,4 +59,14 @@ var (
 		},
 		[]string{},
 	)
+	PodAcknowledgedDuration = opmetrics.NewPrometheusHistogram(
```

Comment:
Should we have an unacknowledged duration as well to mirror this and track when we are hanging?